demo/server.js

```diff
@@ -20,6 +20,7 @@ initSqlJs().then(SQL => {
     sqlstr = "CREATE TABLE model (id INTEGER PRIMARY KEY, name TEXT, address TEXT, description TEXT, model_type TEXT, encoder TEXT, accuracy NUMBER);"
       + "CREATE TABLE data (transaction_hash TEXT PRIMARY KEY, text TEXT);"
       + "CREATE INDEX index_address ON model(address);";
```

Comment:
You'll need to remove the ";" at the end of this line.

Reply:
done !